api/proto/config/devconfig.proto

```diff
@@ -142,6 +147,13 @@ message EdgeDevConfig {
   string project_id = 37;
   string enterprise_name = 38;
   string enterprise_id = 39;
+
+  // Timestamp updated by controller whenever device config is modified.
+  // Used by EVE to properly order configurations that originate at the same controller
+  // but get submitted to device through different ways:
+  //     /config API vs. bootstrap config vs. override.json vs. usb.json etc.
```

Comment:
I don't think we can apply this to override.json/usb.json since they are typically hand-edited.

Reply:
True, override.json/usb.json do have timestamps but it is unlikely that users would fill them correctly. Actually in most cases the timestamp field is not filled in. I will update the comment - The timestamp is still going to be useful, but not in these cases unfortunately.

Reply:
Done (updated comment)